### ISSUE TITLE
Error when launch server

### ISSUE THREAD

**Issue by neohary** [EvEmu-Project/evemu_Crucible#215]
**Describe the bug**
I'm trying to launching server by following the wiki guide, but got an error. I tried run `docker login ` and `docker login quay.io` but no luck.
I never use docker before, so sorry if it's a dumb question.

**To Reproduce**
1. Run `git clone https://github.com/EvEmu-Project/evemu_Crucible.git`
2. Open WLS2, navigate to file location and run `sudo docker-compose -p evemu up --build -d`
3. Error appear ```
[+] Running 0/1
 ⠿ db Error                                                                                                        3.5s
Error response from daemon: unauthorized: access to the requested resource is not authorized
```

**Expected behavior**
Success to launch the server

**Screenshots**
none

**System Details (please complete the following information):**
 - Server OS: Ubuntu 18.04 in WLS2 on Windows 10
 - Docker Desktop 4.7.1 (77678) 
 - EVEmu Version latest commit

**Additional context**
none

**Comment by VizanArkonin** [comment]
That's a recent issue - a docker repository we used to take MariaDB image from has decided to lock themselves behind an auth gate.
A quick solution in this case would be to change the MariaDB image from quay.io to the one from docker-hub
```  db:
    image: "mariadb:10.4"
    container_name: evemu_db
    volumes:
      - db:/bitnami/mariadb
    # If you want to connect to the Database pick your host port to map it to, (host-port:container-port) below. 
    #ports:
    #  - "3306:3306"  
    environment:
      - MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=yes
      - MARIADB_USER=evemu
      - MARIADB_PASSWORD=evemu
      - MARIADB_DATABASE=evemu
volumes:
  db:
  config:
  server_cache:
  image_cache:
```

 The only difference between the two - base image, but from functional standpoint they're about the same.

**Comment by TawfiqulBari** [comment]
Yeah after fixing that, when the server starts building it also tries to pull an image from quay.io/evemu/server:latest
Which doesn't get either. Can you please use a different repo?
![docker-evemu](https://user-images.githubusercontent.com/86520154/168067512-f9f54982-afa2-43f1-bb62-296cbafbcd84.PNG)

**Comment by jdhirst** [comment]
@TawfiqulBari Please build the code using docker, the public docker image is no longer supported.

**Comment by VizanArkonin** [comment]
@TawfiqulBari It looks like you've un-commented the line which allows you to use pre-built image.
`image: "quay.io/evemu/server:latest" #Use prebuilt image`
Try commenting it back and try again - it'll assemble the image on your local machine.

**Comment by jdhirst** [comment]
@TawfiqulBari @neohary I have commited a new change to both staging and master, removing quay.io. Please re-test and close.
